src/main/java/TaskManager.java

```diff
@@ -1,29 +1,38 @@
 public class TaskManager {
+
     private Task[] taskList = new Task[100];
     private int taskCount;
 
     public void addTodoTask(String taskName) {
         taskList[taskCount] = new Todo(taskName);
         taskCount++;
-        Display.displayTaskCreation(taskList[taskCount-1], Display.TODO_TASK_NAME, taskCount);
+        Display.displayTaskCreation(taskList[taskCount-1], Display.TASK_NAME_TODO, taskCount);

```

Comment:
There should be spaces before and after the `-` symbol.

Same issue at lines 22 and 35.

Reply:
Thank you for pointing it out, did not notice it before